src/app/launch-playbook/PlaybookChecklist.tsx

```
@@ -0,0 +1,403 @@
+      }
+      saveChecked(next);
+      return next;
+    });
+  }, []);
+
+  const handleReset = useCallback(() => {
+    const activePhaseData = LAUNCH_PHASES.find((p) => p.id === activePhase);
+    if (!activePhaseData) return;
+    setChecked((prev) => {
+      const next = new Set(prev);
+      activePhaseData.items.forEach((item) => next.delete(item.id));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)